### PR TITLE
added raven and luke

### LINKED ISSUE #18: Add support for more people
# Feature Request

## Is your feature request related to a problem? Please describe:
Hacktoberfest is not held up entirely by just Matt. There are a lot of other amazing people like Raven and Luke who help make hacktoberfest rock!

## Describe the solution you'd like

I would like to add in support to appreciate more people. Whenever someone will mention someone else, It would return the saying with their name.

I would like to implement this myself. Can you please assign it to me?

### PR DESCRIPTION
**Pull request by Atharv-Attri**
# Pull Request

## Description:

Added support for Raven and Luke.

## Related Issue:

Closes #18

### CODE REVIEW

**Comment by Atharv-Attri** [comment]
I'm pretty sure I broke something, IDK what, but if I need to fix anything, please tell me

File: src/phrases.ts [hunk 1]
```diff
@@ -48,3 +48,101 @@ export const matt = [
...
+  "Luke (noun): A Spectacular person!",
+  "Luke (noun): An Incredible person!",
+  "Luke (noun): An Extraordinary person!",
+  "Luke (noun): A Sensational person!",
+  "Luke (noun): A Marvelous person!",
+  "Luke (noun): A Wonderful person!",
+  "Luke (noun): A Thrilling person!",
+  "Luke (noun): A Fantastic person!",
+  "Luke (noun): A Terrific person!",
+];
```
Comment (nhcarrigan) [inline src/phrases.ts:148]:
```suggestion
];

```
New line at the end of the file should fix the lint issue.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
One quick fix, then LGTM.

At some point we should make the phrases unique.

**Comment by Atharv-Attri** [comment]
I fixed it, and Yes, I agree.

**Review by nhcarrigan — APPROVED** [review]
LGTM~! 🥳

**Pull request merged**